WebDriverAgentLib/Categories/XCUIElement+FBFind.m

```diff
@@ -52,6 +52,12 @@ @implementation XCUIElement (FBFind)
   return result.copy;
 }
 
+- (NSInteger) fb_descendantsCountMatchingClassName:(NSString *)className
```

Comment:
There don't seem to be any tests for these changes.

Reply:
I could not find existing tests covering the actual endpoint module (`FBFindElementCommands.m`), so didn't bother with that.

I ran all the tests in Xcode (after first running `Scripts/bootstrap.sh` in the WebDriverAgent root).  It takes several minutes.  A few failed, but look completely unrelated to these changes.

Reply:
OK we can discuss this kind of thing at the retrospective.